server/views/proposedAddress/rejected.pug

```diff
@@ -36,8 +36,8 @@ block content
 
       div.pure-u-1.pure-u-md-1-2.paddingBottom
         p.smallMarginBottom Reason for rejection:
-        if addressReview.consent === 'No'
-          p.bold The homeowner/landlord does not give informed consent
+        if addressReview.consent === 'No' || addressReview.consentHavingSpoken === 'No'
```

Comment:
`consentHavingSpoken` specific to version 2 of curfewAddressReview functionality added to pull through rejection reason content in this latest version.